### ISSUE TITLE
Delegates

### ISSUE THREAD

**Issue by ghost** [mayflower/react-workshop#8]
Wie funktioniert [Event-Delgation](https://javascript.info/event-delegation) mit Javascript?

**Comment by phryneas** [comment]
Im React-Kontext ist das aber irgendwie eher ein Antipattern, oder übersehe ich da was?

**Comment by DirtyHairy** [comment]
Das Eventsystem in React arbeitet mit virtuellen Events und macht afaik bereits Delegation :smirk:

**Comment by phryneas** [comment]
Ja, natürlich - ich meinte jetzt eher, das noch mal innerhalb einer React-Anwendung nachbilden zu wollen und damit Event-Zuständigkeit ohne Kontrolle durch die Kind-Komponenten an Elternkomponenten geben zu wollen wäre eine schlechte Idee.  Und so klingt das Issue für mich ein bisschen, im React-Workshop-Kontext ;)

**Comment by ghost** [comment]
Das Thema wollte @christopherstock gern mal beleuten und zielt im React-Kontext wahrscheinlich auf die Event-Delegation im Framework. Eine eigene Delegation wollten wir nicht nachbilden ;)
